api/hasura/actions/_handlers/updateEpoch.ts

```diff
@@ -146,11 +106,58 @@ async function handler(request: VercelRequest, response: VercelResponse) {
       ],
     },
     {
-      operationName: 'updateEpoch_update',
+      operationName: 'createEpoch_insert',
```

Comment:
@topocount I'm confused by this naming, it seems this is updating an epoch, but the operaton name is create__insert ... 

am I missing something?